src/custom/state/orders/updaters/PendingOrdersUpdater.ts

```diff
@@ -271,12 +275,21 @@ export function PendingOrdersUpdater(): null {
 
       if (!isUpdating.current) {
         isUpdating.current = true
+
+        const allOrders = pendingRef.current.filter((order) => order.class === orderClass)
+        const start = (pageNumber - 1) * LIMIT_ORDERS_PAGE_SIZE
```

Comment:
Very nitpicky but should we have these as utils somewhere to have a unified way of handling pagination instead of inline?